sdk/attestation/attestation/src/utils/typeDeserializer.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ */
+
+import { Serializer, Mapper } from "@azure/core-http";
+
+/**
+ * The TypeDeserializer class enables easy access to the Attestation Model serialization
+ * and deserialization APIs.
+ */
+export class TypeDeserializer {
+
+    public static deserialize<T>(rawJson: any, bodyMapper: Mapper, bodyTypeName: string) : T {
```

Comment:
it is weird to have the return type to be a generic that can not be inferred by anything in the input. In this case, I would use `unknown` instead and let call sites to cast it to the expected type using `as`.